src/models/wc.js

```diff
@@ -0,0 +1,19 @@
+import { round } from "../utilities/utilities.js";
+
+/**
+ * Calculates the Wind Chill Index (WCI) in accordance with the ASHRAE 2017 Handbook Fundamentals - Chapter 9 [18]_.
+ * @param {number} tdb - dry bulb air temperature,[°C]
+ * @param {number} v - wind speed 10m above ground level, [m/s]
+ * @param {object} ifRound - default True, if True rounds output value, if False it does not round it
```

Comment:
need to update this type

Reply:
also [see here](https://github.com/FedericoTartarini/jsthermalcomfort/blob/main/src/models/heat_index.js#L8-L9) for an example of how to annotate these types